ql/src/java/org/apache/hadoop/hive/ql/ddl/table/create/CreateTableOperation.java

```diff
@@ -62,7 +62,8 @@ public int execute() throws HiveException {
     if (desc.getReplicationSpec().isInReplicationScope()) {
       // If in replication scope, we should check if the object we're looking at exists, and if so,
       // trigger replace-mode semantics.
-      Table existingTable = context.getDb().getTable(tbl.getDbName(), tbl.getTableName(), false);
+      Table existingTable = context.getDb().getTable(tbl.getDbName(),
+      tbl.getTableName(), false, true);
```

Comment:
nit: Formatting - I think it is below 120, so we can put it into a single line. If not, then we should indent the second line with 4

Reply:
Fixed